src/main/java/de/uhd/ifi/se/decision/management/jira/persistence/ActiveObjectStrategy.java

```diff
@@ -54,9 +57,11 @@ public Boolean doInTransaction() {
 						} catch (SQLException e) {
 							return false;
 						} finally {
-							for (LinkBetweenDifferentEntitiesEntity linkEntity : ACTIVE_OBJECTS.find(LinkBetweenDifferentEntitiesEntity.class)) {
-								if (linkEntity.getIdOfSourceElement().equals("a"+decisionKnowledgeElement.getId())
-										|| linkEntity.getIdOfDestinationElement().equals("a"+decisionKnowledgeElement.getId())) {
+							for (LinkBetweenDifferentEntitiesEntity linkEntity : ACTIVE_OBJECTS
+									.find(LinkBetweenDifferentEntitiesEntity.class)) {
+								if (linkEntity.getIdOfSourceElement().equals("a" + decisionKnowledgeElement.getId())
```

Comment:
The string literal "a" and similar should be realy constants with nice/telling names.

Reply:
This will be part of future work to get independent of the id prefixes that indicate documentation location.